llvm/lib/Target/AArch64/AArch64SchedNeoverseN2.td

```diff
@@ -858,7 +858,7 @@ def : InstRW<[N2Write_6cyc_1L], (instregex "^LDR[SDQ]l$",
                                            "^LDUR[BHSDQ]i$")>;
 
 // Load vector reg, immed post-index
-def : InstRW<[N2Write_6cyc_1I_1L, WriteI], (instregex "^LDR[BHSDQ]post$")>;
```

Comment:
Thanks, LGTM. Maybe we could merge this with the pre-index form below and use WriteAdr, what do you think?

Reply:
Sure.

Reply:
Done.

Reply:
Thanks, LGTM!